src/libs/ifc/ifcdef.cpp

```diff
@@ -366,6 +367,33 @@ QMap<QString, QString> lineWeightList()
     }
     return map;
 }
+
+//---------------------------------------------------------------------------------------------------------------------
```

Comment:
Please add comments per this guide:
https://developer.lsst.io/cpp/api-docs.html#cpp-doxygen-class-structure